omnipaxos/src/util.rs

```diff
@@ -278,6 +278,8 @@ pub type TrimmedIndex = u64;
 pub type NodeId = u64;
 /// ID for an OmniPaxos configuration (i.e., the set of servers in an OmniPaxos cluster)
 pub type ConfigurationId = u32;
+/// The connectivity of an OmniPaxos node
+pub type Connectivity = u8;
```

Comment:
Since we don't want to make this public and we only used it in BLE, I suggest moving this (without `pub`) to `ballot_leader_election.rs`